eng/Signing.props

```diff
@@ -81,6 +80,10 @@
     <FileSignInfo Include="Valleysoft.DockerCredsProvider.dll" CertificateName="$(ExternalCertificateId)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <FileSignInfo Include="StreamJsonRpc.dll" CertificateName="MicrosoftSHA2" />
```

Comment:
What is this change about?

Reply:
I was getting signing errors in official builds when building the VSIX saying the StreamJsonRpc has wrong certificate specified. I guess it wasn't participating in signing previously, so the mistake went unnoticed. FWIW, the same is specified in roslyn: https://github.com/dotnet/roslyn/blob/c737a04f3e9a895b5b5fff6d4c5467c17e9c49e8/eng/Signing.props#L62

Reply:
Is this a new file you added?  Why did your change cause it?  @marcpopMSFT

Reply:
I think StreamJsonRpc is some dependency brought in because we are building AnalyzerRedirecting as VSIX.